src/Nethermind/Nethermind.JsonRpc/Modules/Trace/TraceRpcModule.cs

```diff
@@ -189,9 +190,21 @@ public ResultWrapper<ParityTxTraceFromStore[]> trace_block(BlockParameter blockP
             return ResultWrapper<ParityTxTraceFromStore[]>.Success(txTraces.SelectMany(ParityTxTraceFromStore.FromTxTrace).ToArray());
         }
 
-        public ResultWrapper<ParityTxTraceFromStore[]> trace_get(Keccak txHash, int[] positions)
+        public ResultWrapper<ParityTxTraceFromStore[]> trace_get(Keccak txHash, long[] positions)
         {
-            throw new NotImplementedException();
+            ResultWrapper<ParityTxTraceFromStore[]> traceTransaction = trace_transaction(txHash);
```

Comment:
Consideration - not sure if implement - keep small cache of recently traced transactions?

Reply:
@LukaszRozmej agree with you, but firstly I think we need to correct trace bugs like: 
https://github.com/NethermindEth/nethermind/issues/3422
https://github.com/NethermindEth/nethermind/issues/3034
https://github.com/NethermindEth/nethermind/issues/3312
https://github.com/NethermindEth/nethermind/issues/3298